### PR TITLE
Installer bugs for 4.14 GA p1

### PR DESCRIPTION
**Pull request by cbippley**
Version(s):
4.14

Issue:
[Bug Batching for 4.14 GA](https://issues.redhat.com/browse/OSDOCS-5987)

Link to docs preview:[
Preview](https://66293--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-bug-fixes)

QE review:
- [ ] QE has approved this change. 
not needed

Additional information:
[Installer bugs](https://issues.redhat.com/secure/Dashboard.jspa?selectPageId=12347491#SIGwKWmOqDAaIiGh1DSCAiYg6oaN4RY8HF6AMAwlQMHEVU1ccdWpPFMCNc1SXIGgGCNeaPAtG0HT+ogCErGsyGEN+BWVIizVlYty2NQcDpLZUdQbgVyIlWeWAOsi2BNUCILgh+5JftSCgaB47qFfMcT8UaPCIpglTgdQGgaM1dQLIU7oxHCD27BaQlWEAA)

### CODE REVIEW

**Review by JoeAldinger — COMMENTED** [review]
A few suggestions.

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -1120,6 +1120,23 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
...
+
+* Previously, the vSphere Terraform `vsphere_virtual_machine` resource did not include the `firmware` parameter. This issue caused the firmware of the virtual machine (VM) to be set to `bios` by default instead of `efi`. Now, the resource includes the `firmware` parameter and sets `efi` as the default value for the parameter, so that the VM runs the Extensible Firmware Interface (EFI) instead of the basic input/output system (BIOS) interface. (https://issues.redhat.com/browse/OCPBUGS-9378[OCPBUGS-9378*) 
+
+* Previously, for clusters that run on {rh-openstack}, in the deprovisioning phase of installation, the installer deleted Object storage containers sequentially. This behavior caused slow and inefficient deletion of objects, especially with large containers. This problem occurred in part because image streams that use Swift containers accumulated objects over time. Now, bulk object deletion now occurs concurrently with up to 3 calls to the {rh-openstack} API, improving efficiency by handling a higher object count per call. This optimization speeds up resource cleanup during deprovisioning. (link:https://issues.redhat.com/browse/OCPBUGS-9081[*OCPBUGS-9081*])
```
[the remark this answers is not in the document]
Comment (cbippley) [reply release_notes/ocp-4-14-release-notes.adoc]: Done

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -1120,6 +1120,23 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
...
+
+* Previously, SSH access to bootstrap and cluster nodes failed when the bastion host ran in the same VPC network as the cluster nodes. Additionally, this configuration caused SSH access from the temporary bootstrap node to the cluster nodes to fail. These issues are now fixed by updating the IBM Cloud `SecurityGroupRules` to support SSH traffic between the temporary bootstrap node and cluster nodes, and to support SSH traffic from a bastion host to cluster nodes on the same VPC network. Log and debug information can be accurately collected for analysis during installer-provisioned infrastructure failure.(link:https://issues.redhat.com/browse/OCPBUGS-8035[*OCPBUGS-8035*]) 
+
+* Previously, DNS records that the installation program created were not removed when uninstalling a private cluster. With this update, the installation program now correctly removes these DNS records. (link:https://issues.redhat.com/browse/OCPBUGS-7973[OCPBUGS-7973]
```
[the remark this answers is not in the document]
Comment (cbippley) [reply release_notes/ocp-4-14-release-notes.adoc]: Fixed

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -1120,6 +1120,23 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
...
+
+* Previously, DNS records that the installation program created were not removed when uninstalling a private cluster. With this update, the installation program now correctly removes these DNS records. (link:https://issues.redhat.com/browse/OCPBUGS-7973[OCPBUGS-7973] 
+
+* Previously, a script provided in the documentation for checking invalid HTTPS certificates in the {rh-openstack} API assumed a recent version of the {rh-openstack} client. For users who did not have a recent version of the client, this script failed. Now, manual instructions are added to the documentation that users can follow to perform the check with any version of the client.
```
[the remark this answers is not in the document]
Comment (cbippley) [reply release_notes/ocp-4-14-release-notes.adoc]: Added

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -1120,6 +1120,23 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
...
+
+* Previously, if you configured user-defined outbound routing when installing a private Azure cluster, the cluster was incorrectly deployed with the default public load balancer. This behavior occurred when using the installer-provisioned infrastructure to install the cluster. With this update, the installation program no longer creates the public load balancer when user-defined routing is configured. (link:https://issues.redhat.com/browse/OCPBUGS-9404[*OCPBUGS-9404*]) 
+
+* Previously, the vSphere Terraform `vsphere_virtual_machine` resource did not include the `firmware` parameter. This issue caused the firmware of the virtual machine (VM) to be set to `bios` by default instead of `efi`. Now, the resource includes the `firmware` parameter and sets `efi` as the default value for the parameter, so that the VM runs the Extensible Firmware Interface (EFI) instead of the basic input/output system (BIOS) interface. (https://issues.redhat.com/browse/OCPBUGS-9378[OCPBUGS-9378*)
```
[the remark this answers is not in the document]
Comment (cbippley) [reply release_notes/ocp-4-14-release-notes.adoc]: done

**Pull request merged**